homeassistant/components/media_player/squeezebox.py

```diff
@@ -293,64 +303,64 @@ def supported_media_commands(self):
 
     def turn_off(self):
```

Comment:
Can these be changed to async equivalents, with `async_query` and `async_update_ha_state`?